### PR TITLE
feat(insights): add docs for creating insights accounts

### PR DESCRIPTION
**Pull request by tyler-dunkel**
### Proposed changes

Adds new docs for the new accounts feature for insights 2.0

### CODE REVIEW

File: content/docs/pulumi-cloud/insights/accounts.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+    parent: insights
+    weight: 3
+---
+
+This document outlines the steps required to create and manage a Pulumi insights account used for scanning provider resources.
+
+## Prerequisites
+
+* You must be an admin of your Pulumi organization.
+* permissions required to create credentials within the provider account you want to scan
```
Comment (interurban) [inline content/docs/pulumi-cloud/insights/accounts.md:23]:
```suggestion
* Permissions required to create credentials within the provider account you want to scan
```

File: content/docs/pulumi-cloud/insights/accounts.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+8. Click `create`. You should see a success notification and arrive on the details page of the account you have created.
+![Account details page](../accounts-details-page.png)
+
+{{< notes type="info" >}}  
+  A scan should be kicked off immediately, if you are creating a new insights account for AWS, see the **child accounts** (one for each region) for scan status.
+{{< /notes >}}
+
+## Account hierarchies
+
+Account hierarchies allow you to organize and manage insights accounts in a structured way. Currently, child accounts can only be created automatically by Pulumi and is only done so in the case of AWS regions. In the future, this feature will be expanded to support creating custom hierarchies, providing more flexibility for structuring accounts, such as for organizing Kubernetes clusters within an Azure subscripiton.
```
Comment (interurban) [inline content/docs/pulumi-cloud/insights/accounts.md:66]:
```suggestion
Account hierarchies allow you to organize and manage insights accounts in a structured way. Currently, child accounts can only be created automatically by Pulumi and is only done so in the case of AWS regions. In the future, this feature will be expanded to support creating custom hierarchies, providing more flexibility for structuring accounts, such as for organizing Kubernetes clusters within an Azure subscription.
```

**Review by interurban — APPROVED** [review]
Adding an approval so you can ship, I'll give it a deeper walk through on monday when I'm back in town.

File: content/docs/pulumi-cloud/insights/accounts.md [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+
+```yaml
+values:
+  environmentVariables:
+    OCI_FINGERPRINT: "25:ad:34:****************:cd:05:05:08:02:a7"
+    OCI_REGION: "us-phoenix-1"
+    OCI_TENANCY_OCID: "ocid1.tenancy.oc1..tenancyidnumbers"
+    OCI_USER_OCID: "user_ocid"
+  files:
+    OCI_PRIVATE_KEY: "PRIVATE_KEY_CONTENT"
```
Comment (rquitales) [inline content/docs/pulumi-cloud/insights/accounts.md:204]: This should be `OCI_PRIVATE_KEY_PATH`. Fix is in #13481

**Pull request merged**